### DISCUSSION TITLE
Matrix-based conditional rules

### DISCUSSION THREAD

**Discussion by rizalgunawan** [vueform/vueform#372] [Questions]
### Discussed in https://github.com/vueform/vueform/discussions/365

<div type='discussions-op-text'>

<sup>Originally posted by **rizalgunawan** November 25, 2024</sup>
How can I implement conditional rules for a matrix component?
Specifically, I want to dynamically change the input type in each row based on the selected value.

<img width="972" alt="image" src="https://github.com/user-attachments/assets/6096d5d2-e38d-4099-ac87-7b457db1b00a">
</div>

**Comment by adamberecz** [comment]
Unfortunately you can't reference cells with conditions, let alone changing the element type.

You might be able to achieve this using a nested List and having two fields for static/dynamic with opposite conditions.
